closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -3101,6 +3134,137 @@ Java_jdk_crypto_jniprovider_NativeCrypto_PBEDerive
     return ret;
 }
 
+/* Create an ECDSA Signature
+ *
+ * Class:     jdk_crypto_jniprovider_NativeCrypto
+ * Method:    ECDSASign
+ * Signature: (J[BI[B)I
+ */
+JNIEXPORT jint JNICALL
+Java_jdk_crypto_jniprovider_NativeCrypto_ECDSASign
+  (JNIEnv *env, jclass obj, jlong key, jbyteArray digest, jint digestLen, jbyteArray sig, jint sigLen)
+{
+    jint ret = -1;
+
+    unsigned char *nativeDigest = NULL;
+    unsigned char *nativeSig = NULL;
+    EC_KEY *privateKey = (EC_KEY*)(intptr_t) key;
```

Comment:
Space before `*` but not after cast:
```c
    EC_KEY *privateKey = (EC_KEY *)(intptr_t)key;
```
also lines 3156, 3170, 3217.

Reply:
Fixed.